### PR TITLE
Dt 1157

### PR DESCRIPTION
**Pull request by preetamnpr**

### CODE REVIEW

File: booking/src/main/java/org/dcsa/conformance/standards/booking/BookingScenarioListBuilder.java [hunk 1]
```diff
@@ -187,28 +188,40 @@ private BookingScenarioListBuilder thenHappyPathFrom(BookingState bookingState)
...
+          uc5_carrier_confirmBookingRequest().thenHappyPathFrom(CONFIRMED, scenarioType)));
+      case RECEIVED -> then(
+        ScenarioType.DG.equals(scenarioType) || ScenarioType.REEFER.equals(scenarioType)?
+        shipper_GetBooking(bookingState).thenEither(
+          uc2_carrier_requestUpdateToBookingRequest().thenHappyPathFrom(PENDING_UPDATE, scenarioType),
+          uc5_carrier_confirmBookingRequest().thenHappyPathFrom(CONFIRMED, scenarioType)):
+          shipper_GetBooking(bookingState).then(
+            uc5_carrier_confirmBookingRequest().thenHappyPathFrom(CONFIRMED, scenarioType))
+        );
+      case START -> then(uc1_shipper_SubmitBookingRequest().thenHappyPathFrom(RECEIVED, scenarioType));
```
Comment (nt-gt) [inline booking/src/main/java/org/dcsa/conformance/standards/booking/BookingScenarioListBuilder.java:224]: Hard to read what is actually happening.

Also, I suspect the condition got inverted based on how scenarios are handled in eBL

Comment (preetamnpr) [reply booking/src/main/java/org/dcsa/conformance/standards/booking/BookingScenarioListBuilder.java:224]: Flavia wanted to have the Update booking for DG and REEFER. Hence I have added this condition. 

✅ SupplyCSP [DG] - UC1 - GET - UC2 - GET - UC3 - GET - UC5 - GET - UC12 - GET
✅ SupplyCSP [DG] - UC1 - GET - UC5 - GET - UC12 - GET

✅ SupplyCSP [REEFER] - UC1 - GET - UC2 - GET - UC3 - GET - UC5 - GET - UC12 - GET
✅ SupplyCSP [REEFER] - UC1 - GET - UC5 - GET - UC12 - GET

Comment (nt-gt) [reply booking/src/main/java/org/dcsa/conformance/standards/booking/BookingScenarioListBuilder.java:224]: I am saying "The code is unreadable" and your answer "Flavia wanted the change". I am 100% certain Flavia did not ask you do write it in the most unreadable way you could, so I feel you are just not answering my review comment.

Comment (preetamnpr) [reply booking/src/main/java/org/dcsa/conformance/standards/booking/BookingScenarioListBuilder.java:224]: I misunderstood ur statement. I got what you mean, probably I will rearrange the condition.

**Review by nt-gt — APPROVED** [review]

**Pull request merged**